NYPLAudiobookToolkit/Core/AudiobookManager.swift

```diff
@@ -182,8 +183,15 @@ var sharedLogHandler: LogHandler?
 extension DefaultAudiobookManager: PlayerDelegate {
     public func player(_ player: Player, didBeginPlaybackOf chapter: ChapterLocation) {
         self.mediaControlHandler.enableMediaControlCommands()
+        if let timer = self.progressSavingTimer {
+          timer.resume()
+        }
+    }
+    public func player(_ player: Player, didStopPlaybackOf chapter: ChapterLocation) {
+        if let timer = self.progressSavingTimer {
```

Comment:
for symmetry with `didBeginPlaybackOf`, should we call `setMediaControlCommands(enabled: false)` here?

Reply:
`setMediaControlCommands(enabled: false)` should be used when we want to completely remove the audiobook from the media control, but here we are only pausing the playback. I think this protocol should be renamed to avoid confusion.

Reply:
oh I see, I didn't know that. That sounds good, or you can put a comment on `setMediaControlCommands(enabled:)` about when that should be used if you prefer.